test/test-tag.js

```diff
@@ -48,3 +48,14 @@ test('allow empty attributes', function (t) {
 
     t.end();
 });
+
+test('should expose getUserAttrs()', function (t) {
+    t.equal(typeof tag.getUserAttrs, 'function');
+    t.end();
+});
+
+test('tag.getUserAttrs() should get and return an object', function (t) {
+    t.throws(tag.getUserAttrs); // eslint-disable-line
```

Comment:
The notice is

> /my_path/caolan.forms/test/test-tag.js
>   58:7  error  .throws is a syntax error  dot-notation
> 
> ✖ 1 problem (1 error, 0 warnings)

I could be fix that with something like this:
```js
t['throws'](tag.getUserAttrs);
```
But for me both look like a workaround. I mean that [`t` has `throws` method](https://github.com/substack/tape#tthrowsfn-expected-msg) and I should have an able to use it as simple as it possible.